pitch.md

```diff
@@ -57,13 +125,38 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+This project is pretty large. I feel like there is a huge amount of work to be
+done in language design. Figuring out how to integrate lots of different methods
+of robot control and allow users to easily specify logic without limiting them
```

Comment:
In an internal DSL, the user should definitely be limited in what they can say in the language but in the host language they should still be able to say anything